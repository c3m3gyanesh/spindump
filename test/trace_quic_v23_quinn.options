--format text --report-notes --report-spins
