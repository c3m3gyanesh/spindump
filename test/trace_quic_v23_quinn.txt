A test between Picoquic client and the Quinn server. Commands that were used:

  ./picoquic/picoquicdemo -l ./test.log -L -a "h3-23" -v ff000017 ralith.com 4433 /5000
