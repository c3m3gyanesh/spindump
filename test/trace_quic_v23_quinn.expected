QUIC 172.16.114.251 <-> 51.15.3.76 08436391f654b544-7b180f8dfde5d7ce (58604:4433) at 1571181687027900 new starting packets 0 0 bytes 0 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 51.15.3.76 08436391f654b544-08b3db9b3d019d7f (58604:4433) at 1571181687340998 measurement up right 313098 packets 1 0 bytes 1280 0 bandwidth 1280 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 51.15.3.76 436391f654b54477-08b3db9b3d019d7f (58604:4433) at 1571181687503550 spinvalue up 1 initiator packets 3 4 bytes 2457 4237 bandwidth 2457 4237 note "V23,no spin"
QUIC 172.16.114.251 <-> 51.15.3.76 436391f654b54477-08b3db9b3d019d7f (58604:4433) at 1571181687503674 spinvalue up 1 initiator packets 4 4 bytes 2540 4237 bandwidth 2540 4237 note "V23,no spin"
QUIC 172.16.114.251 <-> 51.15.3.76 436391f654b54477-08b3db9b3d019d7f (58604:4433) at 1571181687666181 spinvalue up 1 responder packets 5 4 bytes 4008 4237 bandwidth 4008 4237 note "V23,no spin"
QUIC 172.16.114.251 <-> 51.15.3.76 436391f654b54477-08b3db9b3d019d7f (58604:4433) at 1571181687666523 spinvalue up 0 initiator packets 5 5 bytes 4008 4391 bandwidth 4008 4391 note "V23,no spin"
QUIC 172.16.114.251 <-> 51.15.3.76 436391f654b54477-08b3db9b3d019d7f (58604:4433) at 1571181687666524 spinvalue up 0 initiator packets 6 5 bytes 4091 4391 bandwidth 4091 4391 note "V23,no R-spin"
QUIC 172.16.114.251 <-> 51.15.3.76 436391f654b54477-08b3db9b3d019d7f (58604:4433) at 1571181687681404 spinvalue up 1 responder packets 7 5 bytes 5559 4391 bandwidth 5559 4391 note "V23,no R-spin"
QUIC 172.16.114.251 <-> 51.15.3.76 436391f654b54477-08b3db9b3d019d7f (58604:4433) at 1571181687683291 spinvalue up 0 initiator packets 7 6 bytes 5559 4548 bandwidth 5559 4548 note "V23,no R-spin"
QUIC 172.16.114.251 <-> 51.15.3.76 436391f654b54477-08b3db9b3d019d7f (58604:4433) at 1571181687828030 spinvalue up 0 responder packets 8 6 bytes 5625 4548 bandwidth 5625 4548 note "V23,no R-spin"
QUIC 172.16.114.251 <-> 51.15.3.76 436391f654b54477-08b3db9b3d019d7f (58604:4433) at 1571181687828030 measurement up right 161507 packets 8 6 bytes 5625 4548 bandwidth 5625 4548 note "V23,spinning"
QUIC 172.16.114.251 <-> 51.15.3.76 436391f654b54477-08b3db9b3d019d7f (58604:4433) at 1571181687828250 spinvalue up 0 initiator packets 8 7 bytes 5625 4607 bandwidth 5625 4607 note "V23,spinning"
QUIC 172.16.114.251 <-> 51.15.3.76 436391f654b54477-08b3db9b3d019d7f (58604:4433) at 1571181687844054 spinvalue up 0 responder packets 9 7 bytes 5685 4607 bandwidth 5685 4607 note "V23,spinning"
QUIC 172.16.114.251 <-> 51.15.3.76 08cded9b0950d7be-2790995196b7e689 (58211:4433) at 1571181708926748 new starting packets 0 0 bytes 0 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 51.15.3.76 08cded9b0950d7be-08303fdbd0e6d04a (58211:4433) at 1571181709172476 measurement up right 245728 packets 1 0 bytes 1280 0 bandwidth 1280 0 note "No rsp,v23,no spin"
QUIC 172.16.114.251 <-> 51.15.3.76 cded9b0950d7be66-08303fdbd0e6d04a (58211:4433) at 1571181709337895 spinvalue up 1 initiator packets 4 4 bytes 2534 4225 bandwidth 2534 4225 note "V23,no spin"
QUIC 172.16.114.251 <-> 51.15.3.76 cded9b0950d7be66-08303fdbd0e6d04a (58211:4433) at 1571181709338032 spinvalue up 1 initiator packets 5 4 bytes 2617 4225 bandwidth 2617 4225 note "V23,no spin"
QUIC 172.16.114.251 <-> 51.15.3.76 cded9b0950d7be66-08303fdbd0e6d04a (58211:4433) at 1571181709514567 spinvalue up 1 responder packets 6 4 bytes 4085 4225 bandwidth 4085 4225 note "V23,no spin"
QUIC 172.16.114.251 <-> 51.15.3.76 cded9b0950d7be66-08303fdbd0e6d04a (58211:4433) at 1571181709514568 spinvalue up 1 responder packets 6 5 bytes 4085 4379 bandwidth 4085 4379 note "V23,no spin"
QUIC 172.16.114.251 <-> 51.15.3.76 cded9b0950d7be66-08303fdbd0e6d04a (58211:4433) at 1571181709514805 spinvalue up 0 initiator packets 6 6 bytes 4085 4438 bandwidth 4085 4438 note "V23,no spin"
QUIC 172.16.114.251 <-> 51.15.3.76 cded9b0950d7be66-08303fdbd0e6d04a (58211:4433) at 1571181709514839 spinvalue up 0 initiator packets 7 6 bytes 4168 4438 bandwidth 4168 4438 note "V23,no R-spin"
QUIC 172.16.114.251 <-> 51.15.3.76 cded9b0950d7be66-08303fdbd0e6d04a (58211:4433) at 1571181709683156 spinvalue up 0 responder packets 8 6 bytes 5636 4438 bandwidth 5636 4438 note "V23,no R-spin"
QUIC 172.16.114.251 <-> 51.15.3.76 cded9b0950d7be66-08303fdbd0e6d04a (58211:4433) at 1571181709683156 measurement up right 168351 packets 8 6 bytes 5636 4438 bandwidth 5636 4438 note "V23,spinning"
QUIC 172.16.114.251 <-> 51.15.3.76 cded9b0950d7be66-08303fdbd0e6d04a (58211:4433) at 1571181724859735 spinvalue up 1 initiator packets 12 7 bytes 6116 4497 bandwidth 120 4497 note "V23,spinning"
QUIC 172.16.114.251 <-> 51.15.3.76 cded9b0950d7be66-08303fdbd0e6d04a (58211:4433) at 1571181724859735 measurement up left 15176579 packets 12 7 bytes 6116 4497 bandwidth 120 4497 note "V23,spinning"
QUIC 172.16.114.251 <-> 51.15.3.76 cded9b0950d7be66-08303fdbd0e6d04a (58211:4433) at 1571181724859735 measurement up full (initiator) 15344930 packets 12 7 bytes 6116 4497 bandwidth 120 4497 note "V23,spinning"
